jni/CMakeLists.txt

```diff
@@ -267,54 +287,3 @@ if ("${WIN32}" STREQUAL "")
 endif()
 
 # ---------------------------------------------------------------------------
-
-# -------------------------------- INSTALL ----------------------------------
-# Installation rules for shared library
-install(TARGETS ${TARGET_LIBS}
-        LIBRARY DESTINATION lib
```

Comment:
is there a reason why we don't need this?

Reply:
CPack is from cmake tool kit for packaging artifacts - i.e. building RPMs. We added it in ODFE: https://github.com/opendistro-for-elasticsearch/k-NN/issues/122. The idea was to create rpm/deb artifacts for the libraries that then the plugin would depend on for the release process. However, we dont do this anymore. We just package libs in the zip. So I dont see a reason to keep this code around.

Reply:
Thanks for providing the context.